tests/e2e/fixtures/test_data.py

```
@@ -0,0 +1,54 @@
+class TestDataFixtures:
+    """Test data fixtures for E2E testing."""
+    
+    @staticmethod
+    def create_sample_csv():
+        """Create a sample CSV file for testing uploads."""
+        csv_content = """name,age,city
+John Doe,30,New York
+Jane Smith,25,Los Angeles
+Bob Johnson,35,Chicago"""
+        
+        temp_file = tempfile.NamedTemporaryFile(mode='w', suffix='.csv', delete=False)
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)